spec/spec_helper.rb

```diff
@@ -1,5 +1,9 @@
+require 'simplecov'
+SimpleCov.start
+
 $LOAD_PATH.unshift File.expand_path("../../", __FILE__)
 require "rspec-console"
```

Comment:
this is on an undiffed section, so a request to update i assume? sure, i can do.